src/network/taraxa_capability.cpp

```diff
@@ -275,16 +275,16 @@ void TaraxaCapability::interpretCapabilityPacketImpl(NodeID const &_nodeID, unsi
                                    << ", peer PBFT previous round next votes size "
                                    << peer->pbft_previous_round_next_votes_size_;
 
-      auto pbft_synced_period = pbft_chain_->pbftSyncingPeriod();
-      if (pbft_synced_period + 1 < peer->pbft_chain_size_) {
-        LOG(log_nf_) << "Restart PBFT chain syncing. Own synced PBFT at period " << pbft_synced_period
-                     << ", peer PBFT chain size " << peer->pbft_chain_size_;
-        if (pbft_synced_period + 5 < peer->pbft_chain_size_) {
-          restartSyncingPbft(true);
-        } else {
-          restartSyncingPbft(false);
-        }
-      }
+      // auto pbft_synced_period = pbft_chain_->pbftSyncingPeriod();
+      // if (pbft_synced_period + 1 < peer->pbft_chain_size_) {
+      //   LOG(log_nf_) << "Restart PBFT chain syncing. Own synced PBFT at period " << pbft_synced_period
+      //                << ", peer PBFT chain size " << peer->pbft_chain_size_;
+      //   if (pbft_synced_period + 5 < peer->pbft_chain_size_) {
+      //     restartSyncingPbft(true);
+      //   } else {
+      //     restartSyncingPbft(false);
+      //   }
+      // }
```

Comment:
Node will send out status packets every 6 lambda time, and status packet includes node PBFT chain size.
So I think this is a good place to trigger PBFT chain syncing. Otherwise don't need include PBFT/DAG info in status packets